ServerCore/Pages/Responses/CreateBulk.cshtml.cs

```diff
@@ -49,6 +52,14 @@ public async Task<IActionResult> OnPostAsync(int puzzleId)
                 return Page();
             }
 
+            if (DeleteExisting)
+            {
+                Response[] responsesToRemove = await (from Response r in _context.Responses
+                                                      where r.PuzzleID == puzzleId
+                                                      select r).ToArrayAsync();
+                _context.Responses.RemoveRange(responsesToRemove);
```

Comment:
If I recall correctly, removing a response that a team has already submitted breaks a DB constraint. Should this try to fix it or tell the user what's wrong instead of throwing?